packages/http-caching-proxy/src/__tests__/integration/http-caching-proxy.integration.ts

```diff
@@ -26,7 +26,7 @@ describe('HttpCachingProxy', () => {
   let proxy: HttpCachingProxy;
   after(stopProxy);
 
-  beforeEach('clean cache dir', async () => await rimraf(CACHE_DIR));
+  beforeEach('clean cache dir', async () => rimraf(CACHE_DIR));
```

Comment:
Why is this one removed?

Reply:
```ts
async () => await rimraf(CACHE_DIR)
```

The code above is essentially the same as the following:

```ts
(async function () { return await rimraf(CACHE_DIR); }).bind(this)
```

Reply:
Ah, thank you.

Reply:
In this case, `async` is not required either. We can do `beforeEach('clean cache dir', () => rimraf(CACHE_DIR));` as Mocha supports a function that returns a Promise.